dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -261,13 +262,16 @@ def test_categorical_merge(self):
         self.assertIsNotNone(report.pop("times", None))
         report_categories = report['statistics'].pop('categories')
         report_count = report['statistics'].pop('categorical_count')
+        report_gini = report['statistics'].pop('gini_impurity')
         expected_profile = dict(
             categorical=True,
             statistics=dict([
                 ('unique_count', 16),
                 ('unique_ratio', 16 / 1000)
             ]),
         )
+        expected_gini = .914600550
+        self.assertAlmostEqual(report_gini, expected_gini)
```

Comment:
is it possible to get the exact math like above or do we have to do almost equal? if we can get it then we won't need to pop it either.